docs/proposals/mux-logging-service.md

```diff
@@ -0,0 +1,305 @@
+Introduction
```

Comment:
Is this intended to more then a 'proposal'?  Do we expect to debate its introduction and design?  If not, I would move it to live under the 'docs' directory.